win32ss/gdi/eng/lineto.c

```diff
@@ -736,4 +736,117 @@ IntEngPolyline(SURFOBJ *psoDest,
     return ret;
 }
 
+#undef XFORMOBJ
+
+ENGAPI
+BOOL
+APIENTRY
+EngStrokePath(
+    IN SURFOBJ *pso,
+    IN PATHOBJ *ppo,
+    IN CLIPOBJ *pco,
+    IN XFORMOBJ *pxo,
+    IN BRUSHOBJ *pbo,
+    IN POINTL *pptlBrushOrg,
+    IN LINEATTRS *plineattrs,
+    IN MIX mix)
+{
+    // TODO: use pxo, pco
+    // www.osr.com/ddk/graphics/gdifncs_4yaw.htm
```

Comment:
https://web.archive.org/web/20040118234421/www.osr.com/ddk/graphics/gdifncs_4yaw.htm

I have no experience with this code at all, so I doubt that I can give in-depth review.

Reply:
The experienced people was in microsoft. However they don't help us. Can you make some small steps ahead?

Reply:
https://docs.microsoft.com/en-us/windows/desktop/api/winddi/nf-winddi-engstrokepath